### PR TITLE
Initialchanges

### PR DESCRIPTION
**Pull request by wxtim**
- [x] Add unit tests.
- [x] Add GitHub Actions runner. You can see the results on https://github.com/wxtim/cylc-rose/actions

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+[options.entry_points]
+cylc.configure = 
+    rose = cylc.rose.rose:get_rose_vars
```
Comment (wxtim) [inline setup.cfg:19]:
```suggestion
    rose = cylc.rose.rose:get_rose_vars 

```

File: setup.py [hunk 2]
```diff
@@ -24,7 +24,8 @@
 
 INSTALL_REQUIRES = [
     'metomi-rose>=2.0a1',
-    'cylc-flow>=8.0a3'
+    'cylc-flow>=8.0a3',
+    'nest_asyncio'
```
Comment (oliver-sanders) [inline setup.py:28]: Hadn't spotted this earlier, I can't see any `async` statements here yet so I wouldn't have thought that this is necessary?

Comment (wxtim) [reply setup.py:28]: I'll try removing it, but the rose functions involved use asyncio.

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      - name: Configure Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install
+        run: |
+          pip install flake8
+          pip install pytest
+          pip install nest_asyncio
```
Comment (oliver-sanders) [inline .github/workflows/tests.yml:25]: Here too!

File: cylc/rose/rose.py [hunk 4]
```diff
@@ -103,9 +103,6 @@ def rose_fileinstall(dir_=None, opts=None, dest_root=None):
         from metomi.rose.reporter import Reporter
         from metomi.rose.fs_util import FileSystemUtil
 
-        # Allow nested asyncio usage.
-        nest_asyncio.apply()
-
```
Comment (oliver-sanders) [inline cylc/rose/rose.py:108]: Here three.

**Review by oliver-sanders — APPROVED** [review]

**Pull request merged**